### DISCUSSION TITLE
Model calibration.

### DISCUSSION THREAD

**Discussion by kizzle4shaw** [PMEAL/OpenPNM#2730] [Q&A]
How would one calibrate a model from experimental Mercury intrusion porosimetry data?

**Comment by jgostick** [comment]
This is worth writing an example for.  I will try to get to it soon.

**Reply by kizzle4shaw** [reply]
Okay Professor.

**Reply by jgostick** [reply]
How's [this](https://openpnm.org/examples/applications/mercury_intrusion.html)?
